packages/tx/src/eip1559Transaction.ts

```diff
@@ -51,7 +57,7 @@ export class FeeMarketEIP1559Transaction extends BaseTransaction<FeeMarketEIP155
    *
    * @hidden
    */
-  protected DEFAULT_HARDFORK = 'london'
+  protected DEFAULT_HARDFORK = Hardfork.London
```

Comment:
Uhg.

We still have got "London" in here? 🤔

So this is definitely a left over. Just had a look to (re-)get the whole picture, some context: we introduced this "sub"-default HF for tx types which are not yet activated on the current broader default HF (so historically these were actually not "back" but "forward" 😋).

So for 1559 (London) and 2930 (Berlin) we should definitely remove completely.

We can eventually think if we rather want to add for 4844 and set to "Cancun" (so *that* was actually the original intended use case 😋).

On thinking about I would rather think we do not want to do though, seems to have a tendency to get forgotten and think we should be fine with the concrete ´this.common.isActivatedEIP(4844)` capability/EIP checks in the constructor (which is more tx-type-adequate anyhow).

Nevertheless, so: we should remove these two sub-default-HFs before breaking releases!

Reply:
Sure, will push a PR for that!